Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -771,7 +771,7 @@ final class CodecTests: MongoSwiftTestCase {
 
         let rc = ReadConcern(.majority)
         let wc = try WriteConcern(wtimeoutMS: 123)
-        let rp = ReadPreference(.primary)
+        let rp = ReadPreference()
```

Comment:
because we're testing encoding here and not the default read preference behavior, it's probably best to explicitly specify which case we expect in the output. for that reason, I think we can leave this one as-is.